testing/cloud.py

```diff
@@ -47,6 +47,20 @@ def __getattr__(self, name):
         return os.environ[name]
 
 
+def get_resource_path(resource, local_path):
```

Comment:
You can do that with python 3, but not 2. It's invalid syntax to have vargs before kwargs.